src/main/scala/com/lucidchart/open/relate/SqlResult.scala

```diff
@@ -237,6 +237,11 @@ class SqlResult(val resultSet: java.sql.ResultSet) {
     }
   }
 
+  def column[A: ColReader](col: String): A =
+    implicitly[ColReader[A]].forceRead(col, this)
+  def columnOpt[A: ColReader](col: String): Option[A] =
+    implicitly[ColReader[A]].read(col, this)
```

Comment:
What about making these `def apply` and `def opt`? We won't need SqlResults for much else (well, once we separate the "collection of rows" functionality from its "row" functionality).

```scala
sql"".asList { row =>
  row[Int]("id") -> row.opt[Int]("foreign_id")
}
```